tests/sync/test_settings.py

```diff
@@ -1,11 +1,38 @@
+import os
 import pathlib
 
 import pytest
 
 from kitovu import utils
+from kitovu.sync import settings
 from kitovu.sync.settings import Settings, ConnectionSettings
 
 
+@pytest.fixture
+def default_config(monkeypatch, temppath):
+    monkeypatch.setattr(settings.appdirs, 'user_config_dir', lambda _name: str(temppath))
+    return temppath / 'kitovu.yaml'
+
+
+def test_creating_default_config(default_config, monkeypatch):
```

Comment:
The rest of `EditorSpawner` is tested in `test_cli.py`, but this was easier to test here.